### PR TITLE
Unpin from numpy < 1.20

### PR DESCRIPTION
**Pull request by shwina**

### CODE REVIEW

File: ci/gpu/build.sh [hunk 1]
```diff
@@ -83,7 +83,6 @@ gpuci_conda_retry install -y \
 
 # https://docs.rapids.ai/maintainers/depmgmt/
 # gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
-gpuci_conda_retry install -y "numpy<1.20"
```
Comment (kkraus14) [inline ci/gpu/build.sh:86]: There used to be a different text here indicating how to install a package that we should add back in.

Comment (shwina) [reply ci/gpu/build.sh:86]: fixed

**Review by kkraus14 — APPROVED** [review]

**Review by jakirkham — APPROVED** [review]
Thanks Ashwin! 😄

**Review by raydouglass — APPROVED** [review]

**Review by raydouglass — CHANGES_REQUESTED** [review]
I believe this change is also necessary in the `meta.yaml`: https://github.com/rapidsai/cudf/blob/branch-0.18/conda/recipes/cudf/meta.yaml#L41

**Comment by shwina** [comment]
Thanks @raydouglass - fixed!

**Comment by kkraus14** [comment]
@gpucibot merge

**Pull request merged**